woocommerce/woocommerce-android
# [Bug] [IPP Feedback Survey] Add missing dark mode icon

### Description
This PR adds missing dark mode icon to IPP feedback banner.

### Testing instructions
1. Go to Orders screen and verify that the IPP feedback banner has white color in dark mode and black color in light mode.

### Images/gif
| Before | After |
| ------------- | ------------- |
| <img width="401" alt="Screenshot 2023-02-07 at 11 05 27" src="https://user-images.githubusercontent.com/4527432/217221000-7d41b1e8-c1e0-4168-b011-4255ce6af43f.png"> | <img width="399" alt="Screenshot 2023-02-07 at 11 04 24" src="https://user-images.githubusercontent.com/4527432/217221017-a82f23df-9a48-4d5c-9f79-6a843c5ba82e.png">  |
